shells/firefox/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "manifest_version": 2,
+  "name": "React Developer Tools",
+  "description": "Adds React debugging tools to the Chrome Developer Tools.",
+  "version": "2.3.1",
+
+  "applications": {
+    "gecko": {
+      "id": "extension@react.devtools",
+      "strict_min_version": "54.0"
```

Comment:
I should check the versions here.  Things mostly work in 54 and definitely work in 55